strawberryfields/backends/base.py

```diff
@@ -677,3 +680,34 @@ def state(self, modes=None, **kwargs):
             BaseGaussianState: state description
         """
         raise NotImplementedError
+
+
+# ==============================
+# Base bosonic backend
+# ==============================
+
+
+class BaseBosonic(BaseGaussian):
+    """Abstract base class for backends that are only capable of bosonic state manipulation."""
+
+    compiler = "bosonic"
+
+    def __init__(self):
+        super().__init__()
+        self._supported["bosonic"] = True
```

Comment:
Seems like a new option to be added in the docstring of the `BaseBackend.supports` method.

Also, would `bosonic` be the best way to describe operations that the bosonic backend is capable of applying? (E.g., could `gaussian_linear_comb` or something else describe the distinct operations better?)

Reply:
Will modify